assets/schema_input.json

```diff
@@ -29,8 +34,18 @@
                         "maxLength": 0
                     }
                 ]
+            },
+            "library_layout": {
+                "type": "string",
+                "enum": ["single", "paired"],
+                "errorMessage": "library_layout should be only 'single' or 'paired' value"
+            },
+            "library_strategy": {
+                "type": "string",
+                "enum": ["metagenomic", "metatranscriptomic", "genomic", "transcriptomic", "other"],
+                "errorMessage": "library strategy should be only value from list: 'metagenomic', 'metatranscriptomic', 'genomic', 'transcriptomic', 'other'"
             }
         },
-        "required": ["sample", "fastq_1"]
+        "required": ["study_accession","reads_accession", "fastq_1", "library_layout", "library_strategy"]
```

Comment:
Just to confirm, these are all fields that will be received from prefect?

Reply:
To clarify, I see these will be overridden in the miassembler workflow when a samplesheet is used. So, are these still required fields?

Reply:
The idea is either to submit --study and --run-accession OR to use a samplesheet. The values from the samplesheet have the highest priority.